modules/msk/mutect1/meta.yml

```diff
@@ -0,0 +1,57 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "mutect1"
```

Comment:
update the meta.yml documentation, and remove these TODOs. We should be able to get the language from looking at the `help`.

Reply:
a36ee1214bb9a15bfa6797c356d7dc7263f351dc